stack.yaml

```diff
@@ -24,6 +24,8 @@ extra-deps:
   - strict-base-0.4.0.0
   - text-2.0.2
   - yaml-streamly-0.12.2
+  - github: fosskers/versions
+    commit: 7bc3355348aac3510771d4622aff09ac38c9924d
```

Comment:
Temporary until we've confirmed that the fixes are good. Then I'll release `6.0.3` on my end and fix the pin here.

Reply:
We don't actually use stack to build in CI

Reply:
I can't push to your branch, despite github claiming I should, so here:

```diff
commit 94888e9d8edcdde70338d2641b345c2237bf0cb8
Author: Julian Ospald <hasufell@posteo.de>
Date:   Fri Oct 13 17:52:39 2023 +0800

    Add temp git ref to versions to fix CI

diff --git a/cabal.project b/cabal.project
index 2a1fda2..fa6ad49 100644
--- a/cabal.project
+++ b/cabal.project
@@ -8,6 +8,11 @@ package ghcup
 constraints: http-io-streams -brotli,
              any.aeson >= 2.0.1.0
 
+source-repository-package
+  type: git
+  location: https://github.com/fosskers/versions.git
+  tag: 7bc3355348aac3510771d4622aff09ac38c9924d
+
 package libarchive
   flags: -system-libarchive
 
@@ -24,4 +29,4 @@ package streamly
   flags: +use-unliftio
 
 package *
-  test-show-details: direct
\ No newline at end of file
+  test-show-details: direct
diff --git a/cabal.project.release b/cabal.project.release
index b7e3c80..c8c11c4 100644
--- a/cabal.project.release
+++ b/cabal.project.release
@@ -4,6 +4,11 @@ optional-packages: ./vendored/*/*.cabal
 
 optimization: 2
 
+source-repository-package
+  type: git
+  location: https://github.com/fosskers/versions.git
+  tag: 7bc3355348aac3510771d4622aff09ac38c9924d
+
 if os(linux)
   package ghcup
     flags: +tui
```

Reply:
Thanks I'll try that.